### PR TITLE
docs: Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vrslev**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,33 +1,51 @@
...
+# environment variables set.
+docker buildx bake -f docker-bake.hcl *...*
+```
 
-When a PR is sent, the images are built and all commands are tested.
+## Test
 
-If update or fixes to documentation are pushed use `[skip travis]` anywhere in commit message to skip travis.
+### Ping site
+Lightweight test that just checks if site will available after creation.
```
Comment (revant) [inline CONTRIBUTING.md:32]: Lightweight test that just checks if site will **be** available

**Review by revant — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (vrslev) [reply CONTRIBUTING.md:32]:
```suggestion
Lightweight test that just checks if site will be available after creation.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,32 +55,6 @@ Place relevant markdown file(s) in the `docs` directory and index them in README
...
-```
-
-### Test running docker containers
-
-Use the following script
-
-```shell
-./tests/docker-test.sh
-```
+Each Frappe/ERPNext release triggers new stable images builds.
```
[the remark this answers is not in the document]
Comment (vrslev) [reply CONTRIBUTING.md:60]:
```suggestion
Each Frappe/ERPNext release triggers new stable images builds as well as bump to helm chart.
```

**Comment by vrslev** [comment]
@revant Is it good now?

**Review by revant — APPROVED** [review]

**Pull request merged**